graphql2/schema.graphql

```diff
@@ -38,6 +38,12 @@ type Query {
   # Returns a single schedule with the given ID.
   schedule(id: ID!): Schedule
 
+  # Returns the public information of a calendar subscription
+  calendarSubscription(id: ID!): CalendarSubscription
```

Comment:
This was missed during the proposal, but we'll need it for the edit dialog to work. However, it should match contact method query for naming i.e. `userCalendarSubscription`